site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Automate Security Testing with ZAP and GitHub Actions"
+description: "How ZAP baseline and github actions can help to automate the security testing"
+images:
```

Comment:
How about just using a screenshot of the top part of https://github.com/marketplace/actions/owasp-zap-baseline-scan, ie just he logo, and first part of the 'Action Baseline" section?
I'd put this in the blog near the top as well.

Reply:
Done